marklogic-data-hub-central/ui-custom/src/components/SocialMedia/SocialMedia.test.tsx

```diff
@@ -0,0 +1,155 @@
+import {render} from "@testing-library/react";
+import {DetailContext} from "../../store/DetailContext";
+import SocialMedia from "./SocialMedia";
+
+const configMultiple = {
+  component: "SocialMedia",
+  config: {
+    id: "socialmedia",
+    title: "Social Media",
+    site: {
+      arrayPath: "result[0].extracted.person.socials.social",
+      path: "site"
+    },
+    url: {
+      arrayPath: "result[0].extracted.person.socials.social",
+      path: "address"
+    },
+    sites: {
+      facebook: {
```

Comment:
Can we add an example custom size on one of the sites, for completeness? E.g., "size": 24